website/docs/r/vpc_peering_connection.html.markdown

```diff
@@ -123,9 +123,7 @@ the peering connection (a maximum of one).
 must have support for the DNS hostnames enabled. This can be done using the [`enable_dns_hostnames`](vpc.html#enable_dns_hostnames) attribute in the [`aws_vpc`](vpc.html) resource. See [Using DNS with Your VPC](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/vpc-dns.html) user guide for more information.
 
 * `allow_remote_vpc_dns_resolution` - (Optional) Allow a local VPC to resolve public DNS hostnames to
-private IP addresses when queried from instances in the peer VPC. This is
-[not supported](https://docs.aws.amazon.com/vpc/latest/peering/modify-peering-connections.html) for
-inter-region VPC peering.
+private IP addresses when queried from instances in the peer VPC. 
```

Comment:
```suggestion
private IP addresses when queried from instances in the peer VPC.
```
Hi @Elgarni! Can you remove this trailing space to appease our linters? Once that's done we can get this merged!

Reply:
Sure, it's done now

Reply:
@breathingdust PTAL